app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -123,16 +123,13 @@ def job_invocation_params
         job_invocation_params.merge!(job_invocation_params.delete(:ssh)) if job_invocation_params.key?(:ssh)
         job_invocation_params[:inputs] ||= {}
         job_invocation_params[:inputs].permit!
-        job_invocation_params
+        @job_invocation_params = job_invocation_params
```

Comment:
I added this, even if it's unrelated to the PR because this method is inefficiently running every time instead of returning the instance variable (it was not set anywhere)